textproc/fasttext/Portfile

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        facebookresearch fastText 0.9.2 v
+revision            0
+name                [string tolower ${github.project}]
+
+categories-append   textproc
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         Library for fast text representation and classification.
+
+long_description    fastText is a library for efficient learning of \
+                    word representations and sentence classification.
+
+checksums           rmd160  f83305b9e95c7e97a0a51c8a82293b235346315b \
+                    sha256  e9ce4ab7d2bf6c5e2343faa5920c59370d3e8a08bf6bd07b1f1e7ea48d441fe7 \
+                    size    4036928
+
+# Prefers c++14 support
+compiler.cxx_standard   2014
```

Comment:
perhaps requiring the minimum standard (i.e., 2011) is better here since that probably will allow an Apple Clang compiler to be used longer on earlier OSes

Reply:
Thank you. Done.